CMakeLists.txt

```diff
@@ -286,13 +287,26 @@ IF (BALL_HAS_VIEW)
 
 	ADD_DEFINITIONS(${Qt5Widgets_DEFINITIONS})
 
-        LIST(APPEND VIEW_DEP_LIBRARIES Qt5::Widgets
-                                       Qt5::OpenGL
-                                       Qt5::PrintSupport
-                                       Qt5::Network
-                                       Qt5::Test
-                                       Qt5::Xml)
+	LIST(APPEND VIEW_DEP_LIBRARIES Qt5::Widgets
+	                               Qt5::OpenGL
+	                               Qt5::PrintSupport
+	                               Qt5::Network
+	                               Qt5::Test
+	                               Qt5::Xml)
+
+	IF(USE_QTWEBENGINE)
+		FIND_PACKAGE(Qt5WebEngineWidgets REQUIRED)
```

Comment:
Hmm, shouldn't `USE_QTWEBENGINE` rather be disabled then? My line of thought was that, if one wants to `USE_QTWEBENGINE`, both `QtWebEngineWidgets` and `QtWebChannel` are mandatory dependencies. Just like:
```
IF(USE_CUDA)
	FIND_PACKAGE(CUDA REQUIRED)
ENDIF()
```

Ah, and I added you as contributors to my fork as I forgot to push the webengine branch to the main repo before creating the pull request.